src/Analysis/GWASApp/Components/AttritionTableWrapper/AttritionTableModal/AttritionTableModal.jsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Modal } from 'antd';
 import './AttritionTableModal.css';
-import PhenotypeHistogram from '../../../Diagrams/PhenotypeHistogram/PhenotypeHistogram';
-import CohortsOverlapDiagram from '../../../Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram';
+import PhenotypeHistogram from '../../Diagrams/PhenotypeHistogram/PhenotypeHistogram';
+import CohortsOverlapDiagram from '../../Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram';
 
 const AttritionTableModal = ({ modalInfo, setModalInfo }) => {
   const modalWidth = 650;
+  const rowIsOutcome = modalInfo.currentCovariateAndCovariatesFromPrecedingRows?.length === 1;
```

Comment:
Can we use `rowType` from AttritionTableRow instead? I think that would be a more direct and robust way to determine if row is outcome.